ee/billing/quota_limiting.py

```diff
@@ -321,9 +330,10 @@ def org_quota_limited_until(
             "quota_limiting_suspended_until": None,
         }
 
-    # 3. medium / medium high / high trust
-    elif trust_score in [7, 10, 15]:
-        grace_period_days = GRACE_PERIOD_DAYS[trust_score]
+    # 3. medium / medium high / high trust (and feature flags with any trust score)
+    elif trust_score in [7, 10, 15] or resource == QuotaResource.FEATURE_FLAG_REQUESTS:
```

Comment:
```suggestion
    elif trust_score in [7, 10, 15] or minimum_grace_period:
```

Ideally if we do this for a future product we don't want to need to add it here as well. All we care about at this point is that a minimum exists, and what that minimum is.